test/unit/component.test.js

```diff
@@ -43,15 +45,15 @@ QUnit.module('Component', {
   }
 });
 
-const getFakePlayer = function() {
```

Comment:
are the changes in this file necessary?

Reply:
I did these changes since `new Component()`  is expecting a DOM because of

```
if (this.player_) {
    this.on(this.player_, 'languagechange', this.handleLanguagechange);
}
```

in the constructor and  if we use the previews `getFakePlayer()` function  I'm getting
 `Uncaught (in promise) Error: Invalid target; must be a DOM node or evented object`